internal/pkg/config/defaults/gt10000_limits.yml

```diff
@@ -1,6 +1,7 @@
-ram:
-  min: 8192
-  max: 16384
+num_agents:
```

Comment:
Can you link to where these numbers are coming from, best in the PR description?

Reply:
👍 added a link in the description